src/coreclr/vm/jithost.cpp

```diff
@@ -21,33 +21,64 @@ void JitHost::freeMemory(void* block)
     delete [] (BYTE*)block;
 }
 
-int JitHost::getIntConfigValue(const WCHAR* name, int defaultValue)
+int JitHost::getIntConfigValue(const char* name, int defaultValue)
 {
     WRAPPER_NO_CONTRACT;
 
+    SString str(SString::Utf8, name);
```

Comment:
```suggestion
    StackSString str;
    SString(SString::Utf8Literal, name).ConvertToUnicode(str);
```
This is more efficient way to do the conversions (avoids heap allocations and unnecessary copies).